src/nfa/nfa-inclusion.cc

```diff
@@ -225,10 +281,11 @@ bool Mata::Nfa::is_incl(
 { // {{{
     AlgoType algo{ set_algorithm(std::to_string(__func__), params) };
 	return algo(smaller, bigger, alphabet, cex, params);
-} // is_incl }}}
+} // is_included }}}
 
 bool Mata::Nfa::are_equivalent(const Nfa& lhs, const Nfa& rhs, const Alphabet *alphabet, const StringMap& params)
 {
+    //TODO: add comment on what this is doing, what is __func__ ...
     AlgoType algo{ set_algorithm(std::to_string(__func__), params) };
 
     if (params.at("algo") == "naive") {
```

Comment:
Indentation in this file is a big mess. You don't see it because your editor solve it by you but check it here https://github.com/VeriFIT/mata/blob/c6c21c11bc7263b58285f1879ed42a3b1ec695cc/src/nfa/nfa-inclusion.cc
Anyone else would see it like this and it complicates orientation in source file.

Reply:
Yes, I agree. I recommend to never use tabs and some kind of "autotranslation" of tabs to hard spaces.

Reply:
ok